examples/aks-spring-music/porter.yaml

```diff
@@ -1,9 +1,8 @@
 name: spring-music-bundle
 version: 0.1.0
 description: "Spring Music Demo app with Azure Cosmos DB"
-invocationImage: deislabs/spring-music-installer:latest
 dockerfile: Dockerfile.tmpl
-tag: deislabs/spring-music-bundle:v0.1.0
+tag: getporter/spring-music:v0.1.0
```

Comment:
We may want to come up with a naming convention before we publish these to getporter to be consistent. Some of the names use azure-* and others don't for example, and the names don't match the example folder name. Once we push an image, it's there forever and it would be nice to have the names be ones we are happy with long term.

Reply:
I've tried to align the folder name with the bundle name and tag, updating some in the process... let me know if you find any outliers...